EIPS/eip-4762.md

```diff
@@ -243,23 +241,6 @@ Note that values should only be added to the witness if there is sufficient gas
  * this minimum 1/64th gas reservation is checked **AFTER** charging the witness costs when performing a `CALL`, `CODECALL`, `DELEGATECALL` or`STATICCALL`
  * this 1/64th of the gas is subtracted **BEFORE** charging the witness costs when performing a `CREATE` or `CREATE2`
 
-### Replacement for access lists
```

Comment:
accesslists don't need to change structure with the EIP as this info is already available in execution witnesses